src/app/features/tech-record/components/vehicle-technical-record/vehicle-technical-record.component.ts

```diff
@@ -73,17 +74,16 @@ export class VehicleTechnicalRecordComponent implements OnInit, AfterViewInit {
 
     forms.forEach(form => DynamicFormService.updateValidity(form, errors));
 
-    errors.length
-      ? this.errorService.setErrors(errors)
-      : this.errorService.clearErrors();
+    errors.length ? this.errorService.setErrors(errors) : this.errorService.clearErrors();
```

Comment:
The auto-linter at work here again? The new feature introduced does not interact with this code.

Rather than an assignment, this makes calls to 2 separate service methods, having them on separate lines adds clarity. I do not see the benefit of shaving the 2 lines here.